From a510db434151e04a1be4e944e6d740a2ad2f6283 Mon Sep 17 00:00:00 2001
From: Loc Nguyen <loc.nguyen.ks@renesas.com>
Date: Tue, 18 Dec 2018 16:35:29 +0700
Subject: [PATCH] vspm_if_main: Add missing linux header

Add header <linux/of_device.h> to use struct of_device_id

Signed-off-by: Loc Nguyen <loc.nguyen.ks@renesas.com>
---
 vspm_if-module/files/vspm_if/drv/vspm_if_main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/vspm_if-module/files/vspm_if/drv/vspm_if_main.c b/vspm_if-module/files/vspm_if/drv/vspm_if_main.c
index 6989f96..d839080 100644
--- a/vspm_if-module/files/vspm_if/drv/vspm_if_main.c
+++ b/vspm_if-module/files/vspm_if/drv/vspm_if_main.c
@@ -64,6 +64,7 @@
 #include <linux/dma-mapping.h>
 #include <linux/fs.h>
 #include <linux/ioctl.h>
+#include <linux/mod_devicetable.h>
 
 #include "vspm_public.h"
 #include "vspm_if.h"
-- 
2.7.4

